input/pagecontent/overview.md

```diff
@@ -169,7 +169,7 @@ The requestor and owner are restricted to make certain state transitions. For so
 {:.grid .table-hover}
 
 In the first sequence diagram, Care Provider 1 has implemented the (optional) CP-Service role and is requesting Care Provider 2 to do a Task. As a Task MUST always be based on a CarePlan, so if there is none, a CarePlan should be created. 
-> Optional: As Care Provider 2 is updating the Task to 'received', it includes a draft QuestionnaireResponse that Care Provider 1 should complete before a Task can be accepted by Care Provider 2. After Task acceptance, Care Provider 2 is including a second QuestionnaireResponse for patient contact details (which might be needed to send the patient an pre-consult intake Questionnaire).
+> Optional: As Care Provider 2 is evaluating the Task, it may create a new Task (a sub-Task that is based on the first Task) containing a Questionnaire for Care Provider 1. As soon as Care Provider 1 provides the response, Care Provider 2 re-evaluates to first Task. This cycle can be repeated. The reason for cycles of (short, incremental) Questionnaires (or conditional segments in a Questionnaire?) is that if Care Provider 2 evaluates the first response (e.g. inclusion criteria for the Task) it might reject the Task without Practitioner 1 having to answer additional questions.
```

Comment:
"first Task" is ambiguous, "main" Task? And/or we define the term.

Add that the same authorization rules (who may update the subtask) apply as for the main/first Task?

Specify when subtasks can be created/updated for the main/first Task. Or if there's no restriction, specify that.